### PR TITLE
New QR scanner

### PR DESCRIPTION
**Pull request by fdennis**

### CODE REVIEW

File: qml/NewCredentialView.qml [hunk 1]
```diff
@@ -57,16 +60,38 @@ Flickable {
...
-        yubiKey.parseQr(ScreenShot.capture(), function (resp) {
-            scanning = false
-            if (resp.success) {
-                credential = resp
-            } else {
-                navigator.snackBarError(navigator.getErrorMessage(
-                                                                resp.error_id))
-            }
-        })
+        console.log("111")
```
Comment (nevun) [inline qml/NewCredentialView.qml]: 111oneone

Comment (fdennis) [reply qml/NewCredentialView.qml]: I think that is an outdated version of that file.

Comment (nevun) [reply qml/NewCredentialView.qml]: Ah, yes, you are right. Sorry about that :)

/me fetches more coffee

**Review by nevun — COMMENTED** [review]
Would be good to document where and exactly which version of QZXing we pulled in. In case of bug/securitty fixes upstream for example.

**Comment by fdennis** [comment]
> Would be good to document where and exactly which version of QZXing we pulled in. In case of bug/security fixes upstream for example (It seems it is [ftylitak/qzxing@4715b24](https://github.com/ftylitak/qzxing/commit/4715b242cc08582a0c31aad94d74251b77821a29) since the only diff between the pulled in code and upstream is the commit after that.)

You're right! Just pushed a fix for that.

**Pull request merged**